charts/timescaledb-single/templates/statefulset-timescaledb.yaml

```
@@ -462,6 +465,11 @@ spec:
         emptyDir:
           medium: Memory
       {{- end }}
+      - name: post-init
+        projected:
+          defaultMode: 488 # 0750 permissions
```

You *can* use the octal version of modes, which may be more understandable.

```suggestion
          defaultMode: 0750
```